README.rst

```diff
@@ -164,6 +164,16 @@ locked. For example::
         r'\.json$',   # unlock JSON API
     )
 
+LOCKDOWN_REMOTE_ADDR_EXCEPTIONS
+-------------------------------
+An optional list/tuple of IPv4 addresses to be matched agains incomming
```

Comment:
Can you please remove the mention of the tuple and make it a list in the example and the tests? I'd like to keep it in line with Django's `INTERNAL_IPS` which takes a list as well.

Reply:
Ye sure, I can change. But I took example from previous code of tests, e.g. https://github.com/Dunedan/django-lockdown/blob/f659d3a95c8a9e717d2dd80fd075f6b2830eb898/lockdown/tests/tests.py#L50 or https://github.com/Dunedan/django-lockdown/blob/f659d3a95c8a9e717d2dd80fd075f6b2830eb898/lockdown/tests/tests.py#L60 
I want to be consistent with other examples.

Reply:
Yes, the other parameters often promote the ability to use tuples or lists. Going forward I prefer keeping it clean and similar to how Django itself does it and take lists for new parameters. Old parameters of course will keep tuples and lists to keep backward-compatibility.